### ISSUE TITLE
Can't make it run

### ISSUE THREAD

**Issue by Offirmo** [maxlath/wikidata-agent#1]
Following the given instructions

```
git clone git@github.com:maxlath/wikidata-agent.git
npm install
npm start
```

I couldn't make it work.
1. no coffee
2. (after installing coffee) americano module complains about a lot of missing npm modules
3. (after npm installing missing modules) americano complains about a missing config file
4. eventually, it seems the server binds on 127.0.0.1 instead of 0.0.0.0

**Comment by Offirmo** [comment]
Eventually I used https://crossorigin.me/ :-)

**Comment by maxlath** [comment]
some how, I couldn't make it work since a week or two, possibly because of changes on Wikidata auth side, that could have been a problem too. Otherwise, you are welcome to open a PR on missing dependencies and default local ip :)

**Comment by maxlath** [comment]
> no coffee

should have been fixed by b5ff75e

> (after installing coffee) americano module complains about a lot of missing npm modules

should have been fixed by 2b7fbc7

> (after npm installing missing modules) americano complains about a missing config file

@Offirmo could you give more details on that one?

> eventually, it seems the server binds on 127.0.0.1 instead of 0.0.0.0

should have been fixed by 0444c8a
